pkg/otlp/model/translator/metrics_translator_test.go

```diff
@@ -1300,14 +1486,18 @@ func TestMapMetrics(t *testing.T) {
 			if testInstance.instrumentationLibraryMetadataAsTags {
 				options = append(options, WithInstrumentationLibraryMetadataAsTags())
 			}
+			if testInstance.withCountSum {
+				options = append(options, WithCountSumMetrics())
+			}
+
 			tr := newTranslator(t, testLogger, options...)
 			err := tr.MapMetrics(ctx, md, consumer)
 			require.NoError(t, err)
 
 			assert.ElementsMatch(t, consumer.metrics, testInstance.expectedMetrics)
 			assert.ElementsMatch(t, consumer.sketches, testInstance.expectedSketches)
-			assert.Equal(t, observed.FilterMessage("Unknown or unsupported metric type").Len(), testInstance.expectedUnknownMetricType)
-			assert.Equal(t, observed.FilterMessage("Unknown or unsupported aggregation temporality").Len(), testInstance.expectedUnsupportedAggregationTemporality)
+			assert.Equal(t, testInstance.expectedUnknownMetricType, observed.FilterMessage("Unknown or unsupported metric type").Len())
+			assert.Equal(t, testInstance.expectedUnsupportedAggregationTemporality, observed.FilterMessage("Unknown or unsupported aggregation temporality").Len())
```

Comment:
The order of input variables should be `expected` → `actual` according to https://pkg.go.dev/github.com/stretchr/testify/assert#Equal